cmake/configs/default.cmake

```diff
@@ -46,7 +46,7 @@ hunter_default_version(BoringSSL VERSION 0.0.0-0f5ecd3a8-p0)
 hunter_default_version(Box2D VERSION 2.3.1-p0)
 hunter_default_version(CLAPACK VERSION 3.2.1)
 hunter_default_version(CLI11 VERSION 1.8.0)
-hunter_default_version(CURL VERSION 7.60.0-p2)
+hunter_default_version(CURL VERSION 7.74.0-p0)
```

Comment:
These versions don't exist yet - please add them to the `hunter.cmake` files for each project.